docs/developer/quickstart.md

```diff
@@ -107,3 +107,33 @@ make lint
 
 > **Note**
 > fieldalignment errors can be fixed by running: `fieldalignment -fix <path-to-package>`
+
+## Run the Helm Linter
+
+If you have made changes to the Helm Chart, run the following make command from the project's root directory to lint the
```

Comment:
Since the purpose of this guide is to ensure that a new developer can run all the dev commands necessary for opening PRs, can we rephrase this to:
```suggestion
Run the following make command from the project's root directory to lint the
```